index.html

```diff
@@ -613,8 +742,9 @@ <h4>Script Type</h4>
                   They MAY also contain links to audio and mixing instructions for the purpose of producing an audio track incorporating the recordings.</p>
                 <aside class="note">Even though the output of the dubbing workflow is in the same language as those
                 <a>Text</a> objects with the matching language,
-                the <a>Text Language Source</a> of those Text objects remains <a>Translation</a>
-                since they differ from the original language of the dialogue.</aside>
+                the <a>Text Language Source</a> is retained,
+                indicating the source language of those <a>Translation</a> <a>Text</a> objects 
+                since they differ from the untranslated language of the dialogue.</aside>
```

Comment:
I think it isn't worded very well, but it is relevant, and it is related to As-Recorded Scripts.

The potential issue is that after a dubbing recording has been made, someone could philophically decide that the As-Recorded Script should be considered a _transcript_ of the dubbed version instead of a translation of the original version. I think we want to discourage that - do you agree?

Reply:
Will propose a wording change, happy to discuss further.

Reply:
I think I disagree. The "As Recorded Script" is precisely a transcript of what has been recorded in the dubbed audio. Adjustments have been made after the translation from the original version.

Reply:
Upon reading the new text, I understand your point. I'm not sure of the value of the note though. Is there value in knowing that something is not a (pure) transcript or is it purely theoretical? I would still delete the note.